src/uct/base/uct_iface.h

```diff
@@ -688,8 +689,12 @@ uct_iface_invoke_am(uct_base_iface_t *iface, uint8_t id, void *data,
 static UCS_F_ALWAYS_INLINE
 void uct_invoke_completion(uct_completion_t *comp, ucs_status_t status)
 {
-    ucs_trace_func("comp=%p, count=%d, status=%d", comp, comp->count, status);
-    ucs_assertv(comp->count > 0, "comp=%p count=%d", comp, comp->count);
+    ucs_trace_func("comp=%p (%s), count=%d, status=%d", comp,
+                   ucs_debug_get_symbol_name((void*)comp->func), comp->count,
+                   status);
+    ucs_assertv(comp->count > 0, "comp=%p (%s), count=%d, status=%d", comp,
```

Comment:
minor: remove ',' from the format string

Reply:
done